ch_backup/ch_backup.py

```diff
@@ -156,14 +156,17 @@ def backup(
 
         self._context.backup_layout.upload_backup_metadata(self._context.backup_meta)
 
+        skip_lock = self._check_schema_only_backup_skip_lock(sources)
+        logging.debug("Lock will be skipped for backup: {skip_lock}")
```

Comment:
I suggest to move line "Lock will be skipped" after "Starting backup". May be better not here but inside `locker` where we actually skip lock.